src/libraries/System.Net.Quic/tests/FunctionalTests/QuicListenerTests.cs

```diff
@@ -10,7 +10,6 @@ namespace System.Net.Quic.Tests
     [ConditionalClass(typeof(QuicConnection), nameof(QuicConnection.IsQuicSupported))]
     public class QuicListenerTests : MsQuicTestBase
     {
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/32048")]
```

Comment:
This seems similar to asserting that a TCP TLS handshake completes while the connection is still in the TCP backlog, and I'm not sure this is something we should expect. If I slow down accepting connections, I would expect the TLS handshakes to slow down too.

Reply:
That's a good point. We should get a better understanding of how this scenario works.